crypto/bio/bss_dgram.c

```diff
@@ -1914,7 +1914,8 @@ static void get_current_time(struct timeval *t)
     t->tv_sec = (long)(now_ul / 10000000);
     t->tv_usec = ((int)(now_ul % 10000000)) / 10;
 # else
-    gettimeofday(t, NULL);
+    if (gettimeofday(t, NULL) < 0)
+        perror("gettimeofday");
```

Comment:
How ugly! I see the whole bss_dgram using perror() quite liberally. Something to refactor?

Reply:
We should.  The number of `void` functions used makes this painful -- they cannot return an error indication and this seems (slightly) better than having asserts.